.husky/pre-push

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+# Fail fast, exit immediately on any error
+set -e
+
+# Get the name of the current branch
+current_branch=$(git rev-parse --abbrev-ref HEAD)
```

Comment:
Decided to use git rev-parse instead of previous implementation with local_ref because I ran into some issue with local branch that hasn't been published yet and found local_ref isn't always populated, `git  rev-parse` always returns branch name regardless of whether published or not.